charts/umbrella/Chart.yaml

```diff
@@ -109,3 +109,8 @@ dependencies:
     version: 0.1.0
     repository: file://charts/iatpmock
     condition: iatpmock.enabled
+    # ssi-dim-wallet-stub
+  - name: ssi-dim-wallet-stub
+    repository: https://eclipse-tractusx.github.io/charts/dev
+    version: 0.1.1
```

Comment:
```suggestion
    version: 0.1.2
```

Reply:
Workflow triggered for this: https://github.com/eclipse-tractusx/charts/actions/runs/11478310993/job/31942236230

Can you check now?

Reply:
Now it is perfect, thank you